src/utils/apiTranslations.js

```diff
@@ -0,0 +1,13 @@
+const apiTranslations = {
+    en: {
+        // Cron expressions
+        CUSTOM: "Custom",
+        EVERY_DAY_MIDNIGHT: "Every day at midnight",
+        EVERY_DAY_THREE_AM: "Every day at 3 AM",
+        EVERY_HOUR: "Every hour",
+        EVERY_WEEK: "Every week",
+        EVERY_WEEKDAY_NOON: "Every day at noon",
+    },
+};
```

Comment:
I guess that's because those translations where not appearing in i18n/en.pot. That's because the step has this: `displayName: i18n.t(text)`. Unfortunately, we cannot do that, the i18n extractor script expects the literal there. 

Proposal: Write i18n.t(...) in the component. Caveat: AFAIK, you cannot do it in the module definition (or class definition), when the module is loaded, the locale is not set yet. So you have to put it inside a function or similar. It's not trivial to test in this app because we only have EN.

Reply:
Fixed on https://github.com/EyeSeeTea/metadata-synchronization/pull/134/commits/378f9bd18b6c4ea1ffd8763e047c6328049569b0